src/dev/flang/be/c/C.java

```diff
@@ -557,6 +555,14 @@ public CStmnt contract(int cl, FUIR.ContractKind ck, CExpr cc)
   static private final boolean FUZION_DEBUG_TAIL_CALL = "true".equals(System.getenv("FUZION_DEBUG_TAIL_CALL"));
 
 
+  /*
+   * If you want the c-backend to link the JVM,
+   * set this environment variable to e.g.:
+   * FUZION_JVM_PATH=/usr/lib/jvm/java-21-openjdk-amd64
+   */
+  static final String FUZION_JVM_PATH = System.getenv("FUZION_JVM_PATH");
```

Comment:
@fridis @maxteufel Is there another/better way to achieve this than via environment variable?

Reply:
@maxteufel @fridis Yes, I think so too. But if we used `JAVA_HOME` then the jvm would be linked for every fuzion application if `JAVA_HOME` is set. This way linking only happens if this _special_ env variable is set. 

Best solution would probably be if we had a mechanism of determining which intrinsics are used and link only if fuzion.java* is actually used but AFAIK this is currently not possible?

Reply:
changed this